src/main/java/edu/hm/hafner/analysis/parser/CadenceIncisiveParser.java

```diff
@@ -98,10 +97,6 @@ else if (matcher.group(21) != null) {
         if (fileName == null) {
             return Optional.empty();
         }
-        if (fileName.startsWith(SLASH)) {
-            return builder.setFileName(fileName).setLineStart(lineNumber).setCategory(category)
-                    .setMessage(message).setSeverity(priority).buildOptional();
-        }
```

Comment:
I wonder what the original author intended to do here... But anyway, this makes no sense and all tests are green. So it is save to remove.